### ISSUE TITLE
HTML5 elements repeating Landmark roles without ARIA labels

### ISSUE THREAD

**Issue by RachelRVasquez** [10up/Engineering-Best-Practices#151]
We have two `<nav>` elements on the site, each with the landmark role `navigation`. Whenever using a landmark role more than once, an `aria-label` should be added to distinguish between them.

Reference from [the a11y WP handbook](https://make.wordpress.org/themes/handbook/review/accessibility/required/#aria-landmark-roles).

**Comment by RachelRVasquez** [comment]
PR submitted for a fix: https://github.com/10up/Engineering-Best-Practices/pull/152

**Comment by RachelRVasquez** [comment]
Looks like this was merged. Will close this out. 👍
